### ISSUE TITLE
High accuracy on training data(98%) and low accuracy on test data set(60%)

### ISSUE THREAD

**Issue by vidyatshetty** [Microsoft/CNTK#3041]
Hi,

I have a speaker recognition model and predicts correct speakers on training data but fails for testing data set. I worked with dropout technique but nothing helped. Any suggestion would really help.
Thanks

**Comment by KeDengMS** [comment]
Before fighting overfitting, please make sure the test data is preprocessed the same as the training data as the first step. Besides, 100% on training data seems too good to be true, so try increase training data size or use augmentation on training data. Once you got a reasonable big training data and still see overfitting, try using drop_out and/or L1/L2 regularization.

**Comment by vidyatshetty** [comment]
@KeDengMS Thank you for you suggestion I will follow the process you mentioned . On the training set I am getting 98% accuracy that was a typo, sorry for that.
Thanks

**Comment by KeDengMS** [comment]
By preprocessing I mean the generation of htk format files in your case, just to make sure training data and test data shares the same statistics. Usually, training and test data are randomly partitioned from a big data set with proportions like 70%:30%. If you are already doing that, then it's likely a overfitting issue in the model.

**Comment by vidyatshetty** [comment]
@KeDengMS Thank you for writing. As you said it is an overfitting issue. I will work on that.
Thanks
